### PR TITLE
Add header with `clear all`, `export`, & scrolling functionality in Bookmark Modal

### PR DESCRIPTION
**Pull request by felixcodes**
#474 
- Fix overly large React Modal
- Add action and reducer for clearing bookmarks
- Add export bookmarks functionality

### CODE REVIEW

**Comment by felixcodes** [comment]
UI now looks like this :)

image

File: src/reducers/bookmark.ts [hunk 1]
```diff
@@ -52,6 +52,10 @@ export function bookmarkReducer(bookmark: Bookmark, action: Action): Bookmark {
       };
     }
 
+    case BOOKMARK_CLEAR_ALL: {
+      return {...DEFAULT_BOOKMARK};
```
Comment (kanitw) [inline src/reducers/bookmark.ts]: Why not return DEFAULT_BOOKMARK directly?

File: src/components/header/bookmark.scss [hunk 2]
```diff
@@ -15,3 +15,24 @@
   align-items: stretch;
   max-width: 800px;
 }
+
+.modal-header {
```
Comment (kanitw) [inline src/components/header/bookmark.scss]: Is this shared between bookmark and the data loader (if not, no worries).

File: src/components/header/bookmark.scss [hunk 3]
```diff
@@ -14,4 +14,27 @@
   box-align: stretch;
   align-items: stretch;
   max-width: 800px;
+  max-height: calc(100% - 100px);
+  overflow: scroll;
+}
+
+.modal-header {
```
Comment (kanitw) [inline src/components/header/bookmark.scss]: Is this shared between bookmark and the data loader (if not, no worries).

**Comment by kanitw** [comment]
The code looks good -- somehow I have trouble compiling this branch.

@starry97 -- can you help checkout and try `yarn start` on this branch?

**Pull request merged**